dtfs-central-api/package.json

```diff
@@ -54,8 +54,8 @@
     "typescript": "^5.3.2"
   },
   "engines": {
-    "node": ">=18",
-    "npm": ">=9.5.0"
+    "node": ">=21",
```

Comment:
@AlexBramhill, @oscar-richardson-softwire, and I have Node 18 installed via the Company Portal and I've just tried upgrading to Node 21 via the Company Portal and it failed. So I think upgrading the minimum Node version to 21 might cause Alex, Oscar, and I some pain getting our development set ups to work.

If it's urgent then we could just reduce this number on our own laptops until we get higher Node versions installed. What do you think?

Reply:
Hey @lmarrai-sw  thank you for the feedback, I am currently using `v18.15.0` and do not have any impact with higher node version. You should get a warning during docker compose otherwise the development should not be effected.

Please let me know if that is not the case, happy to revert back.

Reply:
Ah you're right it's just a warning. Thanks!